plugins/by-name/yanky/default.nix

```diff
@@ -343,6 +343,14 @@ helpers.neovim-plugin.mkNeovimPlugin {
 
   extraOptions = {
     enableTelescope = mkEnableOption "the `yank_history` telescope picker.";
+
+    sqliteLuaPackage = lib.mkPackageOption pkgs "sqlite-lua" {
```

Comment:
moved to a plugin module?

Reply:
Created https://github.com/nix-community/nixvim/pull/2241 so we can just link to that module's package.